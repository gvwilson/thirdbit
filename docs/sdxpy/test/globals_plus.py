import pprint
my_variable = 123
pprint.pprint(globals())
