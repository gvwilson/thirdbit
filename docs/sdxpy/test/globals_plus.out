{'__annotations__': {},
 '__builtins__': <module 'builtins' (built-in)>,
 '__cached__': None,
 '__doc__': None,
 '__file__': '/sdx/test/globals_plus.py',
 '__loader__': <_frozen_importlib_external.SourceFileLoader object \
at 0x108039290>,
 '__name__': '__main__',
 '__package__': None,
 '__spec__': None,
 'my_variable': 123,
 'pprint': <module 'pprint' from \
'/sdx/conda/envs/sdxpy/lib/python3.11/pprint.py'>}
